routes/bookings.py

```diff
@@ -297,6 +302,31 @@ def patch_booking_by_token(token):
     return '', 204
 
 
+@app.route('/v2/bookings/cancel/token/<token>', methods=["PATCH"])
+@login_or_api_key_required_v2
+def patch_cancel_booking_by_token(token):
+    app_authorization_api_key = extract_api_key_from_request(request)
+    valid_api_key = find_api_key_by_value(app_authorization_api_key)
+
+    booking = booking_queries.find_by(token)
+
+    if current_user.is_authenticated:
+        ensure_current_user_has_rights(
+            RightsType.editor, booking.stock.resolvedOffer.venue.managingOffererId)
```

Comment:
`booking.stock.resolvedOffer.venue.managingOffererId` pourrait être extrait dans une variable 
```suggestion
            booking_offerer = booking.stock.resolvedOffer.venue.managingOffererId
            ensure_current_user_has_rights(RightsType.editor, booking_offerer)
```

Reply:
ou nommé `offerer_id `comme tu l'as fait ligne 318

Reply:
j'ai gardé le offerer_id pour si on veut refactorer après ^^